app/models/concerns/performance_dashboard/overview/enrolled/lot_homeless.rb

```diff
@@ -0,0 +1,59 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module PerformanceDashboard::Overview::Enrolled::LotHomeless
+  extend ActiveSupport::Concern
+
+  # NOTE: always count the most-recently started enrollment within the range
+  def enrolled_by_lot_homeless
+    @enrolled_by_lot_homeless ||= Rails.cache.fetch([self.class.name, cache_slug, __method__], expires_in: 5.minutes) do
+      buckets = lot_homeless_buckets.map { |b| [b, []] }.to_h
+      counted = {}
+      enrolled.preload(:processed_client).order(first_date_in_program: :desc).each do |client|
+        lot = client.processed_client.days_homeless_last_three_years
```

Comment:
Since some clients may not yet have `processed_client` records, we should `next unless client.processed_client` just before setting `lot = `